### PR TITLE
fix(pci.project.kubernetes.details.service): adjust popover placement

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
Set popover placement to auto so that it auto-adjusts for mobile view and does not get clipped

MBP-540

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report web#4506.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

**Comment by ovh-cds** [comment]
CDS Report web#4507.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html [hunk 1]
```diff
@@ -84,8 +84,8 @@
                 <dl class="oui-tile__definition">
                     <dt class="oui-tile__term">
                         <span data-translate="kube_service_file"></span>
-                        <oui-popover>
-                            <button type="button" class="oui-popover-button" oui-popover-trigger></button>
+                        <oui-popover data-oui-popover-placement="auto">
```
Comment (jleveugle) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html]: Why is not the default behavior ? :(

Comment (ganeshkumar1989) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html]: Right now, the default behavior is right. Will discuss with Axel on this :)

Comment (AxelPeter) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html]: The default behaviour was asked `top-center` in the guidelines. Need to see how reacts Popper.js, but we could set it to default `auto` for the next major update (v4).

But here, the preoccupation should be that we should not use `oui-popover` component anymore (deprecated) :p
It should be used as a directive, to help for the migration.

**Pull request merged**